cukinia

```diff
@@ -822,6 +823,32 @@ _cukinia_gpio_sysfs()
 	return 0
 }
 
+# _cukinia_knoerror: Check the kernel has booted without important errors
+# arg1: the message priority
+_cukinia_knoerror()
+{
+	local priority="$1"
+	local ret
+	local use_journalctl
+
+	which journalctl >> /dev/null
+	use_journalctl=$?
+
+	_cukinia_prepare "Checking that kernel has booted with no error (priority: $priority)"
+
+	if [ "$use_journalctl" -eq 0 ]; then
+		journalctl --dmesg --priority "$priority" | grep "No entries"
+		ret=$?
+	else
+		if [[ $(dmesg --level "$priority") || $? -eq 1 ]]; then
```

Comment:
- keep the syntax strictly posix 
- busybox dmesg does not have --level
- this test construct (|| $? -eq 1) is hard to decode, keep it simple